backend/core/migrations/0001_initial.py

```diff
@@ -40,6 +40,6 @@ class Migration(migrations.Migration):
                 ("created", models.DateField(auto_now_add=True)),
                 ("last_updated", models.DateField(auto_now=True)),
             ],
-            options={"abstract": False},
+            options={"abstract": False, "db_table": "core_myuser"},
```

Comment:
this took me a while to figure out. Without this Django will try to create the table with the name `core_user`, which doesn't work